klaviyo/public.py

```diff
@@ -63,8 +72,31 @@ def track(
         if ip_address:
             params['ip'] = ip_address
 
-        query_string = self._build_query_string(params, is_test)
-        return self._public_request(self.TRACK, query_string)
+        if method not in ['get','post']:
+
+            raise ValueError('method argument must be either "post" or "get"')
+
+        else:
+
+            if method == 'post':
+
+                url = '{}/{}'.format(KlaviyoAPI.KLAVIYO_API_SERVER,self.TRACK)
+
+                headers = {
+                    "Accept": "text/html",
+                    "Content-Type": "application/x-www-form-urlencoded"
+                }
+
+                transformed = 'data='+quote(dumps(params))
```

Comment:
Could you rename this variable to something more descriptive? `querystring` maybe? You also might be able to utilize existing methods that exist to transform the params into a query string. Take a look at [`_build_query_string`](https://github.com/klaviyo/python-klaviyo/blob/master/klaviyo/api_helper.py#L122)

Reply:
Good call on querystring. Separately: just looked into _build_query_string, it won't work for this case, as POST does not use base64 encoding. I looked at others, and didn't find anything that could be used, but happy to update if I'm mistaken.

Reply:
updated: renamed to datastring (as these are not in query params), and created a new _build_data_string helper function